### ISSUE TITLE
Cannot get embedded Font Awesome icons to work at all

### ISSUE THREAD

**Issue by egeersoz** [phenixdigital/phoenix_storybook#306]
Elixir 1.14.2
Phoenix 1.7.1
LiveView 0.18.17

We have a lot of components in our app that take a `fa_icon` attribute and render it as part of the component. We would like to have its icons rendered in Storybook's stories as well. We haven't been able to get it to work though.

Example `%Variation{}` for `MyAppWeb.UI.Icon`:

```
%Variation{
  id: :default,
  attributes: %{
    fa_icon: "fa-user",
  }
}
```
This renders the component without the icon:
<img width="814" alt="image" src="https://github.com/phenixdigital/phoenix_storybook/assets/1009235/78972d07-b1e1-4e78-9b27-592982b3f3e4">

Now, `fa-user` is part of the free FA icons. Out of curiosity, I tried changing the `font_awesome_plan` to `:pro` and gave it a `font_awesome_kit_id` (we have confirmed that this kit ID works when the kit is loaded via a <script> tag on the layout). When I did this, the entire storybook page stopped loading. All I see is a white screen with no errors in the browser console or the server terminal.

The browser console only shows these warnings, which are also shown without the FA config changes so I'm not sure if they are related:

```
app.js:20 No storybook configuration detected.
(anonymous) @ app.js:20

app.js:20 If you need to use custom hooks or uploaders, please define them in JS file and declare this     file in your Elixir backend module options (:js_path key).
(anonymous) @ app.js:20
```

**Comment by egeersoz** [comment]
It looks like the issue is with the `lsb-wait-for-icons` class that gets added to `body`:
```
<body class="lsb lsb-max-w-full lsb-overflow-hidden lsb-wait-for-icons" cz-shortcut-listen="true">
```
It never gets removed automatically, but if I remove it manually, everything works.

What triggers the event that removes that class, and why might it not be firing?

**Comment by egeersoz** [comment]
Looking at the source code, it looks like `lsb-wait-for-icons` class gets added to the body if there's a font_awesome_kit_id in the config. There's also a CSS rule for `.fontawesome-i2svg-active .lsb-wait-for-icons` with `display: initial` but there isn't any place where the `fontawesome-i2svg-active` class gets added.

Not sure what the purpose of hiding the entire page to begin with, so I'll submit a PR disabling this entire logic.

**Comment by egeersoz** [comment]
As a temporary workaround, I was able to get things working by adding this to `storybook.css`:
```
.lsb-wait-for-icons {
  display: initial !important;
}
```

**Comment by cblavier** [comment]
Hey Ege, the idea behind `.lsb-wait-for-icons` is to only display the body once all SVG icons have been loaded, it prevents an unpleasant layout shift effect where icons are loaded after the initial content.

https://fontawesome.com/docs/web/dig-deeper/svg-async

The issue might be that you are not using SVG icons? Are you?
If so, I might need an option to enable the  `.lsb-wait-for-icons` only if we are using SVG styling

**Comment by egeersoz** [comment]
@cblavier No, our kit is configured to use web fonts.

**Comment by cblavier** [comment]
I just released `0.5.5` with a new `font_awesome_rendering: :webfont` setting.
Let me know if this is ok for you
